src/hotspot/cpu/s390/macroAssembler_s390.hpp

```diff
@@ -156,7 +156,9 @@ class MacroAssembler: public Assembler {
   unsigned int mul_reg64_const16(Register rval, Register work, int cval);
 
   // Generic operation r1 := r2 + imm.
-  void add2reg(Register r1, int64_t imm, Register r2 = noreg);
+  void add2reg   (Register r1, int64_t imm, Register r2 = noreg);
+  void add2reg_32(Register r1, int64_t imm, Register r2 = noreg);
```

Comment:
I don't understand the difference between these two.  For both, imm must be a simm_32.  I don't think we need add2reg_32.